src/js/player.js

```diff
@@ -349,6 +349,8 @@ class Player extends Component {
       }
     }
 
+    options.audioPosterMode = false;
```

Comment:
suggestion: It seems like this would be better set at the bottom of this file in `Player.prototype.options_`.